### ISSUE TITLE
error trying to pull in moment.js bower component

### ISSUE THREAD

**Issue by jpaas** [josemarluedke/ember-cli-daterangepicker#47]
I'm trying to use 0.0.23 but something is still trying to pull in the moment bower component.

```
The Broccoli Plugin: [SourceMapConcat: Concat: Vendor /assets/vendor.js] failed with:
Error: ENOENT: no such file or directory, stat '/Users/jpaas/Documents/osaaru/imdhealth/admin-webapp/tmp/source_map_concat-input_base_path-kOqJ6y1H.tmp/0/bower_components/moment/moment.js'
```

If I add `ember-cli-moment-shim` to my package.json, it works fine. Its only when I add `ember-cli-daterangepicker`, I get this error.

I've tried cache cleans, and deleting my node_modules and bower_components, but I can't get past this. Any ideas?

**Comment by josemarluedke** [comment]
Hi there! 
You also need to add the following packages to your `package.json`

```
bootstrap-daterangepicker
moment
moment-timezone
```

Let me know if this solves your problem.

**Comment by jpaas** [comment]
@josemarluedke Thanks for the suggestion, but adding those 3 node modules didn't help

**Comment by jpaas** [comment]
I'm explicitly declaring dependencies on:

bootstrap-daterangepicker: 2.1.19
ember-cli-daterangepicker: 0.0.23
ember-cli-moment-shim: 1.1.0
moment: 2.12.0
moment-timezone: 0.5.3

additionally these modules have their own dependencies on "things-moment-related":

```
├─┬ ember-cli-release@0.2.8
│ ├─┬ git-tools@0.1.4
│ │ └── spawnback@1.0.0
│ ├── make-array@0.1.2
│ ├── merge@1.2.0
│ └── moment-timezone@0.3.1
```

```
├─┬ ember-cli-deploy-slack@0.1.0
│ ├── moment-duration-format@1.3.0
```

```
├─┬ ember-cli-deploy-display-revisions@0.1.2
│ └── moment@2.10.6
```

I'm not sure how to figure out who is looking for the moment bower component.

**Comment by kenkogi** [comment]
I have the same issue. 

ember-cli-version: 2.4.2
node: 4.2.4
os: win32 x64

Is there any known fix for this issue?

**Comment by jpaas** [comment]
@kenkogi I never got past this. I ended up using two individual date pickers.

**Comment by josemarluedke** [comment]
Just released version 0.1.0 that moves dependencies to npm. It should fix these problems. Not that you need to include the dependencies to package.json.

Here are the dependencies: https://github.com/josemarluedke/ember-cli-daterangepicker/blob/master/blueprints/ember-cli-daterangepicker/index.js#L8-L11
